content/pl/docs/reference/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Materiały źródłowe
+approvers:
+- chenopis
```

Comment:
Can you remove these lines? We want to use the reviewers/approvers from the OWNERS file instead. Only when the localization will be having a lot of people, files can be appointed to specific contributors.